ui/apps/platform/src/Containers/SystemHealth/DashboardPage.js

```diff
@@ -66,7 +70,18 @@ const SystemHealthDashboardPage = () => {
                     )}
                     {hasReadAccessForAdministration && (
                         <GridItem span={12}>
-                            <VulnerabilityDefinitionsHealthCard pollingCount={pollingCountSlower} />
+                            <VulnerabilityDefinitionsHealthCard
+                                component="SCANNER"
+                                pollingCount={pollingCountSlower}
+                            />
+                        </GridItem>
+                    )}
+                    {hasAccessForScannerV4 && hasReadAccessForAdministration && (
```

Comment:
@RTann @dcaravel I just wanted to double-check if the logic for showing this card is correct. We will check if the feature flag is on and has `Administration` read access (same as above)

Reply:
That's accurate per my understanding, confirmed that the API to pull the vuln defs update requires `Administration` access:

https://github.com/stackrox/stackrox/blob/03b5f817f0087480b29c796db2b243d327281144/central/integrationhealth/service/service_impl.go#L28-L30